src/components/Layout/index.jsx

```diff
@@ -22,7 +23,9 @@ function Layout(props) {
           </div>
         </div>
       </header>
+      <main id="layout-main">
       <main id="layout-main">{props.children}</main>
+      </main>
```

Comment:
We have 2 issues here:

1. We have two elements with the same ID, you should use the `id` attribute for a single element.
2. We have 2 `<main>` tags which is invalid, you must have a single one.

```suggestion
      <main id="layout-main">
          {props.children}
      </main>
```